Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM gcc:9.3.0
+WORKDIR gambit_install
+COPY . .
```

Comment:
`COPY . . ` is noop?

Reply:
Nope! It copies the local gambit files into the dockerfile

Reply:
Ah, yes, the source and destination aren't on the same machine.

Reply:
Otherwise, the dockerfile only contains a "virgin" ubuntu install.

You could replace `COPY . .` by `RUN git clone https://github.com/gambit/gambit.git`, but I think that you can save the network I/O since you already have all the gambit files locally (aka if you have this Dockerfile, I assume you cloned gambit and are running the commands inside the gambit folder)

Reply:
> Ah, yes, the source and destination aren't on the same machine.

exactly :)